modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -30,29 +30,86 @@ async fn fetch_unique_licenses(ctx: &TrustifyContext) -> Result<(), anyhow::Erro
+        "CDDL-1.0 OR LicenseRef-GPL-2.0-with-classpath-exception".to_string(),
+        "CDDL-1.1 OR LicenseRef-GPL-2.0-with-classpath-exception".to_string(),
         "EPL-1.0".to_string(),
-        "GPL-2.0-only".to_string(),
+        "GPL-2.0-only AND MIT".to_string(),
         "ISC".to_string(),
-        "JSON".to_string(),
-        "LGPL-3.0-or-later".to_string(),
+        "ISC AND JSON".to_string(),
+        "ISC AND MIT".to_string(),
+        "JSON AND MIT".to_string(),
+        "LGPL-3.0-or-later OR Apache-2.0".to_string(),
+        "LicenseRef-0".to_string(),
+        "LicenseRef-0 AND LicenseRef-4".to_string(),
+        "LicenseRef-0 AND LicenseRef-8".to_string(),
+        "LicenseRef-0 AND LicenseRef-BSD".to_string(),
+        "LicenseRef-0 OR LicenseRef-10".to_string(),
+        "LicenseRef-1".to_string(),
+        "LicenseRef-10 AND LicenseRef-6 AND LicenseRef-GFDL".to_string(),
+        "LicenseRef-11".to_string(),
+        "LicenseRef-12".to_string(),
+        "LicenseRef-13".to_string(),
+        "LicenseRef-14".to_string(),
+        "LicenseRef-2".to_string(),
+        "LicenseRef-2 AND LicenseRef-BSD".to_string(),
+        "LicenseRef-3".to_string(),
+        "LicenseRef-4 OR LicenseRef-Artistic".to_string(),
+        "LicenseRef-6".to_string(),
+        "LicenseRef-6 AND (LicenseRef-0 OR LicenseRef-10)".to_string(),
+        "LicenseRef-6 AND LicenseRef-GFDL AND LicenseRef-BSD AND MIT".to_string(),
+        "LicenseRef-6 OR LicenseRef-BSD".to_string(),
+        "LicenseRef-7 AND (LicenseRef-Artistic OR LicenseRef-4)".to_string(),
+        "LicenseRef-8".to_string(),
+        "LicenseRef-8 AND LicenseRef-0 AND LicenseRef-6".to_string(),
+        "LicenseRef-8 AND LicenseRef-6".to_string(),
+        "LicenseRef-8 AND MIT AND LicenseRef-0".to_string(),
+        "LicenseRef-9".to_string(),
+        "LicenseRef-AFL AND LicenseRef-0".to_string(),
+        "LicenseRef-BSD".to_string(),
+        "LicenseRef-BSD AND LicenseRef-8".to_string(),
+        "LicenseRef-BSD AND LicenseRef-GPLv2".to_string(),
+        "LicenseRef-BSD OR LicenseRef-4".to_string(),
+        "LicenseRef-GPLv2".to_string(),
+        "LicenseRef-GPLv2 AND LicenseRef-0 AND LicenseRef-LGPLv2 AND MIT".to_string(),
+        "LicenseRef-GPLv3".to_string(),
+        "LicenseRef-JasPer".to_string(),
+        "LicenseRef-LGPLv2".to_string(),
+        "LicenseRef-LGPLv2 OR LicenseRef-MPLv1.1".to_string(),
         "MIT".to_string(),
-        "MPL-1.0".to_string(),
+        "MIT AND LicenseRef-2 AND LicenseRef-CC-BY AND LicenseRef-GPLv3".to_string(),
+        "MIT AND MPL-1.0".to_string(),
+        "MIT AND WTFPL".to_string(),
         "MPL-2.0".to_string(),
         "NOASSERTION".to_string(),
         "Python-2.0".to_string(),
-        "Sendmail".to_string(),
-        "Unlicense".to_string(),
-        "WTFPL".to_string(),
         "Zlib".to_string(),
+        "Zlib AND LicenseRef-Boost".to_string(),
+        "Zlib AND Sendmail AND LicenseRef-8".to_string(),
```

Comment:
I think you can safely remove all the `.to_string()` calls in here, and your assertion will still work.

Reply:
done